src/domain/occurrences/queries/OccurrenceQueries.ts

```diff
@@ -1,8 +1,8 @@
 import gql from 'graphql-tag';
 
 export const occurrencesQuery = gql`
-  query Occurrences($eventId: String) {
-    occurrences(eventId: $eventId) {
+  query Occurrences($projectId: ID, $eventId: String) {
+    occurrences(projectId: $projectId, eventId: $eventId) {
```

Comment:
Hmm, that is a good question 🙂  In practice probably not, but at least in theory it might be possible there comes a request for occurrences for an event that is not part of the current project. Maybe having multiple tabs open with one outdated having the previous project selected, or something.

Those are of course rare corner cases, and as said most probably there won't be issues in practice, but then again adding the project filter does no harm what so ever, so I prefer still adding it to make sure the thing is more robust.